runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -230,11 +230,11 @@ def SanityCheck(): bool
     err = 'Cannot debug, +channel feature is not supported'
   elseif (way is Way.Prompt) && !exists('*prompt_setprompt')
     err = 'Cannot debug, missing prompt buffer support'
-  elseif (way is Way.Prompt) && !empty(glob(gdb_cmd))
+  elseif (way is Way.Prompt) && !empty(glob($'./{gdb_cmd}')) || !empty(glob($'./{gdb_cmd}/'))
```

Comment:
should we use `getcwd()` instead of `.` here?

Reply:
Yes.

Reply:
Done!